MCprep_addon/optimize_scene.py

```diff
@@ -31,6 +31,13 @@
 CMP_BOUNCES = MIN_BOUNCES * 2
 MAX_FILTER_GLOSSY = 1.0
 MAX_STEPS = 200
+MIN_SCRAMBLING_MULTIPLIER = 0.35
```

Comment:
For these any any other above constants, if you have any one-line comment to add to why these values are set, would be greatly helpful.